packages/orbit-components/src/TextLink/__tests__/index.test.js

```diff
@@ -25,6 +25,17 @@ describe("#TextLink", () => {
     expect(onClick).toHaveBeenCalled();
   });
 
+  it("should execute onClick method once when stopPropagation", () => {
+    const onClick = jest.fn();
+    render(
+      <TextLink onClick={onClick} stopPropagation>
+        {title}
+      </TextLink>,
+    );
+    userEvent.click(screen.getByText(title));
+    expect(onClick).toHaveBeenCalledTimes(1);
```

Comment:
not a big deal, but in that case `toHaveBeenCalled` is enough i think :)

Reply:
@mainframev this PR fixes the bug where the handler was being called more than once. Using `toHaveBeenCalled` would pass even if the bug was still present, so using `toHaveBeenCalledTimes(1)` is the right thing to do.